web/packages/teleport/src/generateResourcePath.ts

```diff
@@ -39,7 +42,6 @@ export default function generateResourcePath(
     .replace(':startKey?', params.startKey || '')
     .replace(':query?', processedParams.query || '')
     .replace(':search?', processedParams.search || '')
-    .replace(':searchAsRoles?', processedParams.searchAsRoles || '')
```

Comment:
Is this removal intentional?

Reply:
No, was accidentally removed while porting over the changes from my previous refactor PR. Good catch